### ISSUE TITLE
How to mark TEI:sex/@value for Eunuchs

### ISSUE THREAD

**Issue by davidamichelson** [srophe/srophe-app-data#682]
This issue is split from issue #660

**Comment by davidamichelson** [comment]
There are two records for Eunuchs at present:
1148.xml
1255.xml

I think we should mark them also as originally F or M. 

This would mean having multiple values for sex. What do you think?

**Comment by davidamichelson** [comment]
@wsalesky How would this affect faceting on sex?

**Comment by davidamichelson** [comment]
A third Eunuch to add would be 

http://wwwb.library.vanderbilt.edu/exist/apps/srophe/person/1584, which is currently :
<sex xml:lang="en" value="F E" source="#bib1584-3">female, but thought by her associates to be a eunuch</sex>
It's a nice description but it messes up the faceting. We could do
<sex xml:lang="en" value="F" source="#bib1584-3">female</sex>
<sex xml:lang="en" value="E" source="#bib1584-3">eunuch</sex>
and leave it up to the hagiography description to give the explanation.

**Comment by wsalesky** [comment]
It will simplify the display but will have no effect on the code.

**Comment by davidamichelson** [comment]
@wsalesky Would this work:

```<sex xml:lang="en" value="F" source="#bib1584-3">female</sex>
<sex xml:lang="en" value="E" source="#bib1584-3">eunuch</sex>```

**Comment by wsalesky** [comment]
Yes, that will be fine.

**Comment by dlschwartz** [comment]
@davidamichelson, I don't think this is right. The guidelines reserve sex for biological sex, not gender. See discussion here: http://www.tei-c.org/release/doc/tei-p5-doc/en/html/ND.html#NDPERSbp. In SPEAR, I use trait type="gender" ref=".../keyword/eunuch".

**Comment by wlpotter** [comment]
@jsaintlauren FYI, https://syriaca.org/person/1148 and https://syriaca.org/person/1255 are listed as eunuchs (with `<sex value="E">eunuch</sex>`. Person 1584 (Anastasia, cf. #683 ) is still listed as `<sex xml:lang="en" value="F">female</sex>`

@davidamichelson @dlschwartz do we need to discuss whether we should be using tei:sex for eunuchs, per @dlschwartz's comment just above?
